docs/paper/admin/reference/configuration/server-properties.mdx

```diff
@@ -0,0 +1,224 @@
+---
+slug: /reference/server-properties
+---
+
+# Server Properties
+
+import React from 'react';
+import Config from '@site/src/components/Config';
+
+### This page documents the properties that can be set in the `server.properties` file.
+
+:::info
+
+Click on a property to learn more about it.
+
+:::
+
+<Config data={`
+allow-flight:
+  default: 'false'
+  description: 'Means that users will not be kicked if they fly whilst in Survival mode. This is likely to occur through
+    hacking however there can be false positives.'
+allow-nether:
+  default: 'true'
+  description: 'Allows users to travel to the Nether.'
+broadcast-console-to-ops:
+  default: 'true'
+  description: 'Send console command output to all online operators.'
+broadcast-rcon-to-ops:
+  default: 'true'
+  description: 'Send rcon command output to all online operators.'
+debug:
+  default: 'false'
+  description: 'Enables the server''s debug mode.'
+difficulty:
+  default: 'easy'
+  description: 'Defines the difficulty of the server. (Allowed values: "peaceful", "easy", "normal", "hard")'
+enable-command-block:
+  default: 'true'
+  description: 'Enables command blocks.'
+enable-jmx-monitoring:
+  default: 'false'
+  description: 'Exposes an MBean with the Object name "net.minecraft.server:type=Server" and two attributes
+    "averageTickTime" and "tickTimes" exposing the tick times in milliseconds. In order for enabling JMX on
+    the Java runtime you also need to add a couple of JVM flags to the startup as documented
+    [here](https://docs.oracle.com/javase/8/docs/technotes/guides/management/agent.html)'
+enable-query:
+  default: 'false'
+  description: 'Enables GameSpy4 protocol server listener. Used to get information about server.'
+enable-rcon:
+  default: 'false'
+  description: 'Enables remote access to the server console.'
+enable-status:
+  default: 'true'
+  description: 'Makes the server appear on the server list and also enables the query listener to get server
+    information. If turned off, server will appear offline but players will still be able to connect.'
```

Comment:
could someone get confused that
since enable-status `enables the query listener`, it overrides enable-query?

Reply:
I suppose it’s possible. I’ll take a look at the rest of your comments later. Thanks for the review